cli/tests/integrations/test_task.py

```diff
@@ -257,6 +257,30 @@ def test_ls_bad_path():
         returncode=1)
 
 
+def test_ls_completed():
+    # create a completed task
+    with app(SLEEP_COMPLETED, 'test-app-completed'):
+        # get its task id
+        task_id_completed = _get_completed_task_id()
+
```

Comment:
ok, than we should change the name of this app so we don't create two apps with the same name which could cause unexpected behavior.